### PR TITLE
[Notifier][PropertyInfo][VarDumper] Remove unused code

### PR DESCRIPTION
**Pull request by alamirault**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Issues        | Fix #... 
| License       | MIT

Remove unused code

### CODE REVIEW

File: src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php [hunk 1]
```diff
@@ -688,10 +688,6 @@ private function getPropertyName(string $methodName, array $reflectionProperties
      */
     private function findAdderAndRemover(\ReflectionClass $reflClass, array $singulars): array
     {
-        if (!\is_array($this->arrayMutatorPrefixes) && 2 !== \count($this->arrayMutatorPrefixes)) {
```
Comment (nicolas-grekas) [inline src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php:691]: why don't we need the `2 !== \count($this->arrayMutatorPrefixes)` check anymore?

Comment (stof) [reply src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php:691]: being a public static method, I would keep the safeguard here as there is nothing avoiding mutating it, even though it is internal.

**Review by OskarStark — APPROVED** [review]
Please check fabbot, thanks

LGTM for the Notifier part

File: src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php:691]: IIUC, this means this change should be reverted, isn't it @alamirault?

Comment (alamirault) [reply src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php:691]: I reverted `2 !== \count($this->arrayMutatorPrefixes)` part

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @alamirault.

**Pull request merged**